glide-core/src/protobuf/redis_request.proto

```diff
@@ -135,6 +135,7 @@ enum RequestType {
     Rename = 91;
     DBSize = 92;
     Brpop = 93;
+    ZMScore = 94;
```

Comment:
94 is already occupied by a PR on upstream => you will have a conflict
Pick another number. 99 is the next free AFAIK